Update upgrade steps documentation (#2172)
owncloud/docs: modules/admin_manual/pages/maintenance/upgrading/upgrade_php.adoc

= Upgrade PHP on RedHat 7 and CentOS 7
:toc: right
:toclevels: 1
:keywords: upgrade, red hat, centos
:description: Upgrade PHP to versions 7.0 - 7.3 on Red Hat and CentOS so that you can make the most out of your ownCloud installations.

== Introduction

You should, almost, always upgrade to the latest version of PHP supported by ownCloud, if and where possible. 
And if you're on a version of PHP older than 5.6 you *need* to upgrade. 
This guide steps you through upgrading your installation of PHP to version 7.0, 7.1, 7.2 or 7.3 on Red Hat or CentOS 7.

:from-version: 5.6

> :to-version: 7.0

Reviewer: 7.0 & 7.1 are deprecated as well. You should at least install PHP 7.2

Reviewer 2: 10.3.2 still works with PHP 7.0: https://github.com/owncloud/core/blob/v10.3.2/console.php#L38 - I guess the next minor server version will definitely drop PHP 7.0 support?

Reviewer 2: yes, master ot is https://github.com/owncloud/core/blob/master/console.php#L38

Author: I'm all for upgrading to the most recent version, but was erring on the slower side, in consideration for users who may not be in a position to upgrade so far yet. However, encouraging 7.2 makes sense. Maybe I'm being a little too conservative here.

Author: Taking a middle ground and removing 7.0.

Author: The 10.3.2 release lists PHP PHP 7.0.8 as the minimum required version. To me, then, it doesn't make sense to remove 7.0 yet.

= Upgrade PHP on RedHat 7 and CentOS 7
:toc: right
:toclevels: 1
:keywords: upgrade, red hat, centos

> :description: Upgrade PHP to versions 7.0 - 7.4 on Red Hat and CentOS so that you can make the most out of your ownCloud installations.

Reviewer 3: ```suggestion
:description: Upgrade PHP to versions 7.0 - 7.3 on Red Hat and CentOS so that you can make the most out of your ownCloud installations.
```
PHP 7.4 support by ownCloud is not "a thing" yet.

== Introduction

> You should, almost, always upgrade to the latest version of PHP, if and where possible. 

Reviewer 3: ```suggestion
You should, almost, always upgrade to the latest version of PHP supported by ownCloud, if and where possible. 
```
(because, e.g., the current latest PHP is 7.4, but that is not yet supported by ownCloud

Author: Thanks for adding this one. Really helps to avoid any confusion.

And if you're on a version of PHP older than 5.6 you *need* to upgrade. 

> This guide steps you through upgrading your installation of PHP to version 5.6 or 7.0 on Red Hat or CentOS 7.

Reviewer 3: ```suggestion
This guide steps you through upgrading your installation of PHP to version 7.0, 7.1, 7.2 or 7.3 on Red Hat or CentOS 7.
```

Author: Thanks for catching that one.
